### PR TITLE
Indexer

### PR DESCRIPTION
**Pull request by ProgrammingSR**
Added DutyCycle changes
Added feeder changes
Added Tia changes

### CODE REVIEW

**Review by ArnoUUU — APPROVED** [review]

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -76,9 +76,9 @@
 	}
 
 	public static final class IntakeConstants {
-		// TODO update angles with intake
-		public static final double kRaisedArmAngle = 0;
-		public static final double kLoweredArmAngle = 0;
+		// TODO: update speeds
+		public static final double kRaiseArmSpeed = 0.6;
+		public static final double kLowerArmSpeed = -0.3;
```
Comment (LennyAbbas) [inline src/main/java/frc/robot/Constants.java:81]: why speed?

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -91,4 +91,7 @@
 		public static final int kDriverControllerPort = 0;
 		public static final double kControllerDeadband = 0.11;
 	}
+	public static final class ConversionConstants {
+		public static final short kDegreesInARotation = 360;
+	}
```
Comment (LennyAbbas) [inline src/main/java/frc/robot/Constants.java:96]: constants are for storing numbers specific to our robot not the degrees in a circle

File: src/main/java/frc/robot/DutyCycleAbsoluteEncoder.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DutyCycle;
+import frc.robot.Constants.ConversionConstants;
+
+public class DutyCycleAbsoluteEncoder {
```
Comment (LennyAbbas) [inline src/main/java/frc/robot/DutyCycleAbsoluteEncoder.java:11]: we should only create our own encoder wrapper as a last resort, I think DutyCycleEncoder will work for us

File: vendordeps/REV2mDistanceSensor.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
+{
```
Comment (LennyAbbas) [inline vendordeps/REV2mDistanceSensor.json:1]: why would we use a distance sensor when we already have the color sensor that can determine distance

**Review by LennyAbbas — DISMISSED** [review]

**Comment by LennyAbbas** [comment]
The PR needs to be renamed to something descriptive

**Review by thetaspirit — APPROVED** [review]

**Pull request merged**